Update single-sign-on-sso.md (#28193)
fleetdm/fleet: docs/Deploy/single-sign-on-sso.md

`Applies only to Fleet Premium`

> Fleet automates user creation using Just-in-time (JIT) provisioning. Fleet uses System for Cross-domain Identity Management (SCIM) to [map end users' identity provider (IdP) information to host vitals](https://fleetdm.com/guides/foreign-vitals-map-idp-users-to-hosts). SCIM for user provisioning is coming soon.

Reviewer: ```suggestion
Fleet automates user creation using just-in-time (JIT) provisioning. Fleet uses System for Cross-domain Identity Management (SCIM) to [map end users' identity provider (IdP) information to host vitals](https://fleetdm.com/guides/foreign-vitals-map-idp-users-to-hosts). SCIM for user provisioning is coming soon.
```

`Applies only to Fleet Premium`

> Fleet supports two ways to automate user creation: Just-in-time (JIT) provisioning and SCIM-based provisioning.
> 
> This section explains how JIT user provisioning works. With JIT, Fleet will automatically create a user account when someone logs in for the first time using your configured SSO. This removes the need to create individual user accounts for a large organization.
> 
> > **Note:** If you're using SCIM for automated user provisioning and updates, that setup happens through your identity provider and is separate from JIT. For more details on SCIM-based provisioning, refer to your IdP’s SCIM documentation.

Reviewer 2: ```suggestion
Fleet automates user creation using Just-in-time (JIT) provisioning. Fleet uses System for Cross-domain Identity Management (SCIM) to [add end user's identity provider (IdP) information to host vitals](https://fleetdm.com/guides/foreign-vitals-map-idp-users-to-hosts). SCIM for user provisioning is coming soon.

This section explains how JIT user provisioning works. With JIT, Fleet will automatically create a user account when someone logs in for the first time using your configured SSO. This removes the need to create individual user accounts for a large organization.
```

@mike-j-thomas currently, Fleet only supports JIT provisioning. Fleet uses SCIM for human-device mapping (guide will be published along with 4.68, PR [here](https://github.com/fleetdm/fleet/pull/26989/files)) not using provisioning yet.

cc @marko-lisica

> When JIT user provisioning is turned on, Fleet will automatically create an account when a user logs in for the first time with the configured SSO. This removes the need to create individual user accounts for a large organization.

Author: ```suggestion
When JIT user provisioning is turned on, Fleet will automatically create an account when a user logs in for the first time with the configured SSO.
```
